modules/otel-forwarding-traces.adoc

```diff
@@ -39,12 +39,18 @@ metadata:
 rules:
   # <1>
   # <2>
-- apiGroups: ["", "config.openshift.io"]
-  resources: ["pods", "namespaces", "infrastructures", "infrastructures/status"]
+- apiGroups: [""]
+  resources: ["pods", namespaces",]
+  verbs: ["get", "watch", "list"]
+- apiGroups: ["apps"]
+  resources: ["replicasets"]
+  verbs: ["get", "watch", "list"]
+- apiGroups: ["config.openshift.io"]
+  resources: ["infrastructures", "infrastructures/status"]
```

Comment:
@pavolloffay, the two callouts below are added as extra lines **40** and **41**. Would it make sense to move `# <1>` and `# <2>` to lines **46** and **49** respectively?

Reply:
yes, I am not sure why it was done that way initially